### DISCUSSION TITLE
Is it open source

### DISCUSSION THREAD

**Discussion by fazalhakim426** [PDFTron/pdftron-sign-app#14] [Q&A]
I need the same on my website. can I integrate the code into my Laravel site?

**Comment by mparizeau-pdftron** [comment]
Hi, you can see the license here https://github.com/PDFTron/pdftron-sign-app/blob/master/LICENSE

> WebViewer React UI project/codebase or any derived works is only permitted in solutions with an active commercial PDFTron WebViewer license.

You can find out more about licensing here https://www.pdftron.com/pricing/
